src/models/bonw15/internal-overview.md

```diff
@@ -0,0 +1,27 @@
+# Bonobo WS (Internal Overview)
+
+## Bottom components and connectors:
+
+![Internal Components](./img/components-highlighted.webp)
+
+- LCD panel connector is highlighted in light green
+- Webcam/microphone connector is highlighted in cyan
+- Keyboard adhesive access point is highlighted in pink
+- Wireless card (under the innermost SSD) is highlighted in navy
```

Comment:
I think the "(under the innermost SSD)" note is left over from another model

Reply:
Fixed with https://github.com/system76/tech-docs/pull/176/commits/e2544d2cbcd090bbbcdbb6a1b396b9d7facd85d3.